packages/config/config/devices/0x0234/matrix_zba7140.json

```diff
@@ -0,0 +1,634 @@
+// Logic Group MATRIX ZBA7140
+// Matrix ZBA7140
+{
+	"revision": 4,
+	"manufacturer": "Logic Group",
+	"manufacturerId": "0x0234",
+	"label": "MATRIX ZBA7140",
+	"description": "Matrix ZBA7140",
+	"devices": [
+		{
+			"productType": "0x0004",
+			"productId": "0x0129"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"maxNodes": 5,
+			"isLifeline": true
+		},
+		"2": {
+			"label": "Button 1 Basic",
+			"description": "Button 1 Basic",
+			"maxNodes": 5
+		},
+		"3": {
+			"label": "Button 1 Multilevel",
+			"description": "Button 1 Multilevel",
+			"maxNodes": 5
+		},
+		"4": {
+			"label": "Button 2 Basic",
+			"description": "Button 2 Basic",
+			"maxNodes": 5
+		},
+		"5": {
+			"label": "Button 4 Basic",
+			"description": "Button 4 Basic",
+			"maxNodes": 5
+		},
+		"6": {
+			"label": "Button 3 Basic",
+			"description": "Button 3 Basic",
+			"maxNodes": 5
+		},
+		"7": {
+			"label": "Button 3 Multilevel",
+			"description": "Button 3 Multilevel",
+			"maxNodes": 5
+		},
+		"9": {
+			"label": "Button 4 Multilevel",
+			"description": "Button 4 Multilevel",
+			"maxNodes": 5
+		}
+	},
+	"paramInformation": {
+		"1": {
+			"label": "Enable Central Scene notifications",
+			"description": "This parameter can be used for enabling or disabling Central scene notifications",
```

Comment:
Doesn't add much value, remove

Reply:
I dont want to remove a parameter from the actual device from the descriptions from manufacturer. But i agree to remove description

Reply:
That's what I meant - I thought you can see which lines I'm referring to.

Reply:
![image](https://user-images.githubusercontent.com/33527165/111026660-b7415f00-83eb-11eb-9bd3-531304bc7550.png)

It kinda shows everything, but i get it now, the description needs to be remove because of clutter :)